reana_client/api/client.py

```diff
@@ -127,6 +127,26 @@ def create_workflow(self, user, organization, reana_spec):
         except Exception as e:
             raise e
 
+    def start_analysis(self, user, organization, workflow):
+        """Start a workflow."""
+        try:
+            (response,
+             http_response) = self._client.api.set_analysis_status(
+                                  user=user,
+                                  organization=organization,
+                                  analysis_id=workflow,
+                                  status='start').result()
+            if http_response.status_code == 200:
+                return response
+            else:
+                raise Exception(
+                    "Expected status code 200 but replied with "
+                    "{status_code}".format(
+                        status_code=http_response.status_code))
+
+        except Exception as e:
```

Comment:
Regarding exceptions https://github.com/reanahub/reana-server/issues/18

Reply:
I suppose there is no need for additional status code handling here, exceptions are delivered to the client as they come from reana-server